src/pages/settings/Payments/PaymentMethodList.js

```diff
@@ -151,23 +151,6 @@ class PaymentMethodList extends Component {
             });
         }
 
-        if (!this.props.shouldShowAddPaymentMethodButton) {
-            return combinedPaymentMethods;
-        }
-
-        combinedPaymentMethods.push({
```

Comment:
Because we're removing this, we would also have to remove the check from `renderItem`, `if (item.type === BUTTON) {`